ui/src/components/common/Header.css

```diff
@@ -1,5 +1,6 @@
-.mssHeader {
+.pf-c-page__header{
```

Comment:
@laurafitzgerald I wonder if using the pf internal classes is the right way to go - can you expand on your reasoning for the change?

Reply:
Good point @damienomurchu I second that. We should not modify the internal classes, but add a second class to override some of the values.